stories/ContentPlaceholder.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { number, withKnobs } from '@storybook/addon-knobs';
+import { ContentPlaceholder } from '@textkernel/oneui';
+
+storiesOf('ContentPlaceholder', module)
+    .addDecorator(withKnobs)
+    .add('ContentPlaceholder', () => {
+        const duration = number('Animation duration in seconds', null);
+        const height = number('Custom height in pixels', null);
+
+        return (
+            <div style={{ width: number('Container width in pixels', 400) }}>
+                <div style={{ margin: '6px 0', lineHeight: '12px' }}>
+                    <ContentPlaceholder
+                        duration={duration}
+                        height={height}
+                        width={number('Width for placeholder 1', null)}
+                    />
+                </div>
+                <div style={{ margin: '6px 0', lineHeight: '14px' }}>
+                    <ContentPlaceholder
+                        duration={duration}
+                        height={height}
+                        width={number('Width for placeholder 2', 55)}
+                    />
+                </div>
+                <div style={{ margin: '6px 0', lineHeight: '16px' }}>
+                    <ContentPlaceholder
+                        duration={duration}
+                        height={height}
+                        width={number('Width for placeholder 3', 65)}
+                    />
+                </div>
+            </div>
+        );
```

Comment:
I think this structure will result in a not very beautiful html when rendered, with many nested divs. Can you give example that is simpler? Like adding the `<ContentPlaceholder>` without wrapping each in an individual `div`. E.g. add a classname so that they get the margins they need?
Also wrapping them in `p` or `h1` might be more intuitive when trying to mimic a look, if wrapping is necessary. There is a problem with that though, since the placeholder itself is a `div`, one cannot nest it in `p`. The solution might be to change the placeholder to render `span` instead.
What do you think?

Reply:
I'm not sure how to produce a simpler DOM structure than having it render two elements while preserving the required functionality. It shouldn't use `span`, because spans are not meant to be block-level elements - this requires a block-level display.
I think you just shouldn't use them inside a paragraph or heading (doesn't make much sense); just show the placeholder instead of the heading, for example?

Reply:
yes. Probably the best would be if we don't need to wrap each placeholder at all, apart from the main parent element, which sets the width. but can we add the spacing otherwise? Maybe even by default by adding a rule for `ContentPlaceholder + ContentPlaceholder`?
Of course, one can still do the div thing as here, but that if you want to do fancier things. It would be nice if something like this would work out of the box:
```
<div style={width: '400px'}>
<ContnetPlaceholder/>
<ContnetPlaceholder/>
<ContnetPlaceholder/>
</div>
```

Reply:
Would it be better to apply the spacing on application level, since it depends quite much on how you wish it to be implemented? Otherwise I suppose we would have to implement various levels of spacing, making assumptions on how the placeholder is used. I'd say the most convenient thing to do is to pass an additional classname of your own and have the spacing be defined in the application CSS. What do you say?